todo/tests/integration/test_task_detail_api.py

```
@@ -1,76 +1,51 @@
-from unittest.mock import patch
-from rest_framework import status
-from rest_framework.test import APITestCase
-from django.urls import reverse
 from bson import ObjectId
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)